web-api/src/business/useCases/caseDeadline/createCaseDeadlineInteractor.ts

```diff
@@ -43,13 +43,15 @@ export const createCaseDeadline = async (
       hasCaseDeadline: true,
     });
 
-  await applicationContext.getUseCaseHelpers().updateCaseAndAssociations({
-    applicationContext,
-    authorizedUser,
-    caseToUpdate: caseEntity,
-  });
+  const result = await applicationContext
+    .getUseCaseHelpers()
+    .updateCaseAndAssociations({
+      applicationContext,
+      authorizedUser,
+      caseToUpdate: caseEntity,
+    });
 
-  return newCaseDeadline;
+  return new Case(result, { authorizedUser }).validate().toRawObject();
```

Comment:
This mimics what `deleteCaseDeadlineInteractor.ts` does, and the response from createCaseDeadlineInteractor was never really used.